code401challenges/ArrayReverse.java

```diff
@@ -7,6 +7,12 @@ public static void main(String[] args) {
   }
 
   public static int[] reverseArray(int[] arr) {
+    for (int i = 0; i < arr.length/2; i++){
+      int temp = arr[i];
+      arr[i] = arr[arr.length -1 - i];

```

Comment:
Be sure to maintain consistency in your code. If you're not going to include a space after arithmetic operators then make sure you don't have a space after any arithmetic operators (this code still works perfect!):
`arr.length -1 -i` instead of `arr.length -1 - i`